File: GameRanksClient/src/app/models/entity/UserPageReviewStruct.ts
import {Review} from "./Review";

export class UserPageReviewStruct{
	review: Review;
	gameId: number;
	gameName: string;
	gameImage: string;

	constructor(review: Review, gameId: number, gameName: string, gameImage: string){
		this.review = review;
		this.gameId = gameId;
		this.gameName = gameName;
		this.gameImage = gameImage;
	}
}

File: GameRanks/src/main/java/GameRanks/GameRanks/clientStruct/element/UserPageReviewStruct.java
package GameRanks.GameRanks.clientStruct.element;

import GameRanks.GameRanks.model.Review;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class UserPageReviewStruct {
    private Review review;
    private long gameId;
    private String gameName;
    private String gameImage;
}
